chapter1/poisson.ipynb

```diff
@@ -257,7 +257,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "d829e410",
+   "id": "827c3e69-77ac-4312-a4dd-a1c26a40b27c",
```

Comment:
I think that this id can be changed by hand to its original value using a text editor (or from the GitHub web interface). If it causes a problem, our CI build will spot it.